### PR TITLE
Adds storybook to core package.

### PR DESCRIPTION
**Pull request by ionab**
This pull request introduces the Config for Storybook to the core packages and starts storybooking the components.

To run the storybook navigate to the core package and run 
`yarn storybook`.

The storybook can be stopped at anytime using Ctl + C.
To build the storybook for deployment, use `yarn build-storybook`

### CODE REVIEW

File: packages/core/.storybook/babel-ts.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  test: /\.(tsx?)|(js)$/,
+  exclude: /(node_modules)/,
+  use: {
+    loader: require.resolve('babel-loader'),
+    options: {
+      presets: [[require('@fesk/babel-config'), { typescript: true }]],
```
Comment (stephenwf) [inline packages/core/.storybook/babel-ts.js]: Is `babel-ts.js` part of storybook? If not, I don't think this file is being used (dupl. in `config.js`)

File: packages/core/src/components/ImageCropper/ImageCropper.stories.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+storiesOf('ImageCropper', module)
+  .addDecorator(withKnobs)
+  .add('ImageCropper', () => {
+    return (
+      <ImageCropper
+        imgHeight={300}
+        imgWidth={200}
+        //TODO get accurate URL for this component
+        iiifUrl={'https://wellcomelibrary.org/iiif/b18035723/'}
```
Comment (stephenwf) [inline packages/core/src/components/ImageCropper/ImageCropper.stories.js]: ```https://dlcs.io/thumbs/wellcome/1/ff2085d5-a9c7-412e-9dbe-dda87712228d/info.json```
might work

Comment (ionab) [reply packages/core/src/components/ImageCropper/ImageCropper.stories.js]: That's brilliant, worked perfectly!

File: packages/core/.storybook/babel-ts.js (hunk 1 above)
Comment (ionab) [reply packages/core/.storybook/babel-ts.js]: Good spot, I've removed this file as not required.

**Pull request closed without merging**